packages/website/src/components/Navbar/BaseNavbar.tsx

```diff
@@ -65,3 +45,74 @@ export const DesktopBaseNavbar = () => {
     </>
   );
 };
+
+export const MobileBaseNavbar = () => {
+  const [isVisible, setIsVisible] = useState(false);
+  const logoLayout = useBreakpointValue({ sm: 'square', md: 'horizontal' } as const);
+  const { route } = useRouter();
+  const iconButonSize = 'md';
```

Comment:
💭 I'm not sure if there is a need for this constant. In theory menu open and close buttons are separate components and doesn't have to share the same size